IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CommunicationsApplianceTypeAutomaton/DocPropertySet.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CommunicationsApplianceTypeAutomaton" UniqueId="ac8ba82e-d39e-0184-a856-fa356c131fe6" ApplicableType="IfcCommunicationsAppliance/AUTOMATON" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Ha missed the "i" this time! `Pset_CommunicationsApplianceTypeAutomation`

Reply:
It is AUTOMATON: https://www.electropedia.org/iev/iev.nsf/display?openform&ievref=351-42-32

Reply:
Defined as a PDT here: https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC4-voting/HTML/

Reply:
Ahhh! How Sci fi!